job_templates/packaging_job.xml.em

```diff
@@ -134,7 +134,7 @@ if [ -n "${CI_AMENT_TEST_ARGS+x}" ]; then
   if [[ "$CI_AMENT_TEST_ARGS" != *-- ]]; then
     CI_AMENT_TEST_ARGS="$CI_AMENT_TEST_ARGS --"
   fi
-  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS"
+  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS --"
```

Comment:
I was incorrect in saying that l134 adds the delimter beforehand. 
I'll investigate why it's not appending `--` as expected at L134 (as seen in http://ci.ros2.org/job/test_packaging_linux/3/console)
the difference to the non-packing jobs is that the `--ros1-path` arg gets passed afterwards.